addons/hr_holidays/views/hr_leave_accrual_views.xml

```diff
@@ -87,14 +87,17 @@
                             <field name="name"/>
                         </group>
                         <group>
+                            <field name="company_id" invisible="1"/>
```

Comment:
No need to put invisible field, because we already have the field in the view :innocent:

Reply:
But the present company_id field is not available unless you're in the base.group_multi_company group, so I think it still is required

Reply:
You are right :innocent: